src/styles/calendar.scss

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: calc($day-size * 7 + $gap-size * 6);
```

Comment:
The variable `$day-size` is used but not defined in this file. Ensure it is defined or imported from another SCSS file.